src/main/java/serverutils/aurora/mc/AuroraMinecraftHandler.java

```diff
@@ -0,0 +1,74 @@
+package serverutils.aurora.mc;
+
+import java.util.Arrays;
+import java.util.HashSet;
+
+import cpw.mods.fml.common.Loader;
+import cpw.mods.fml.common.ModContainer;
+import cpw.mods.fml.common.eventhandler.SubscribeEvent;
+import serverutils.aurora.AuroraConfig;
+import serverutils.aurora.AuroraHomePageEvent;
+import serverutils.aurora.AuroraPageEvent;
+import serverutils.aurora.page.HomePageEntry;
+
+public class AuroraMinecraftHandler {
+
+    public static final AuroraMinecraftHandler INST = new AuroraMinecraftHandler();
+
+    @SubscribeEvent
+    public void onHomeEvent(AuroraHomePageEvent event) {
+        event.add(new HomePageEntry("Mod List", "modlist", "https://i.imgur.com/yQNrfg7.png"));
```

Comment:
Can we embed these images or pull from the minecraft assets so that its not reliant on imgur?

Reply:
I would love that, but I don't have a clue how to embed a local image onto an html page using java...

Reply:
oh right, I was in the aurora embed

Reply:
The image is of Redstone dust so it's not like we lose anything major if Imgur goes down later